src/main/java/org/spongepowered/api/entity/player/Player.java

```diff
@@ -118,4 +122,93 @@
      */
     void clearTitle();
 
+    /**
+     * Gets the current value for the given {@link Statistic}.
+     *
+     * @param statistic The statistic to return
+     * @return The current value for the given statistic, if a value is set yet.
+     *         Optional.absent() otherwise
+     */
+    Optional<Long> getStatistic(Statistic statistic);
+
+    /**
+     * Gets all {@link Statistic}s with their current values. Does not return
+     * statistics that does not have a value set yet.
+     *
+     * @return An immutable map containing all statistics with their current
+     *         values
+     */
+    Map<Statistic, Long> getStatistics();
+
+    /**
+     * Adds the specified amount to the given statistic.
+     *
+     * @param statistic The statistic to update
+     * @param amount The amount to add to the statistic
+     */
+    void addToStatistic(Statistic statistic, long amount);
+
+    /**
+     * Sets the given statistic to the given value.
+     *
+     * @param statistic The statistic to update
+     * @param newValue The new value for the statistic
+     */
+    void setStatistic(Statistic statistic, long newValue);
+
+    /**
+     * Resets the given statistic. The entry will be removed from the stats so
+     * if you try to get it, it will return Optional.absent() until it is
+     * set again.
+     *
+     * @param statistic The statistic to reset
+     */
+    void resetStatistic(Statistic statistic);
+
+    /**
+     * Resets all statistics. The entries will be removed from the stats so if
+     * you try to get any of them, it will return Optional.absent() until they
+     * are set again.
+     */
+    void resetStatistics();
+
+    /**
+     * Checks whether this player has earned the given {@link Achievement}.
+     *
+     * @param achievement The achievement to check
+     * @return True if the player has completely earned the achievement
+     */
+    boolean hasAchievement(Achievement achievement);
+
+    /**
+     * Gets an {@link Iterable} containing all {@link Achievement}s this player
+     * has earned already.
+     *
+     * @return An immutable iterable containing all earned achievements
+     */
+    Iterable<Achievement> getAchievements();
```

Comment:
Its part of not enforcing anything by the API.
Because a `Set` may also be a valid choice for a data type here.
But you are somehow right, i will change it to a `Collection` to allow the usage of `contains()`.